pkg/steps/output_image_tag.go

```diff
@@ -68,15 +68,15 @@ func (s *outputImageTagStep) Run(ctx context.Context, dry bool) error {
 		fmt.Printf("%s\n", istJSON)
 	} else {
 		if err := s.istClient.ImageStreamTags(toNamespace).Delete(ist.Name, nil); err != nil && !errors.IsNotFound(err) {
-			return err
+			return fmt.Errorf("could not remove output imagestreamtag: %v", err)
 		}
 		_, err := s.istClient.ImageStreamTags(toNamespace).Create(ist)
 		if errors.IsAlreadyExists(err) {
 			// another job raced with us, but the end
 			// result will be the same so we don't care
 			return nil
 		}
-		return err
+		return fmt.Errorf("could not create output imagestreamtag: %v", err)
```

Comment:
This is incorrect, err can be nil.  I'll fix this instance, but I'll need you to verify the other cases aren't like this.

Reply:
Hmmmmm. I'll have to look through the rest tomorrow